runner/ansible/roles/1.4.1/tasks/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.4.1'
+
+- name: "{{ test_id }}.check"
+  shell: |
+    # if no Azure Fence Agent exit, with a pass
+    cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/@type" > /dev/null 2>&1 || exit 0
+    # Check that following match expected, exit with error if any do not
+    [[ $(cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/instance_attributes/nvpair[@name='pcmk_monitor_retries']" | grep -oP 'value="\K[^"]+') != "{{ expected[test_id + '.pcmk_monitor_retries'] }}" ]] && exit 1
+    [[ $(cibadmin -Q --xpath "//primitive[@type='fence_azure_arm']/instance_attributes/nvpair[@name='pcmk_action_limit']" | grep -oP 'value="\K[^"]+') != "{{ expected[test_id + '.pcmk_action_limit'] }}" ]] && exit 1
+    [[ $(cibadmin -Q --xpath "//git ststusprimitive[@type='fence_azure_arm']/instance_attributes/nvpair[@name='power_timeout']" | grep -oP 'value="\K[^"]+') != "{{ expected[test_id + '.power_timeout'] }}" ]] && exit 1
```

Comment:
I believe this `git ststus` was miss-typed here.

Reply:
Besides this typo, the check looks good though

Reply:
Thanks